### PR TITLE
add linux installers to build process

### PR DESCRIPTION
**Pull request by josephsmendoza**
The product name was lowercased to conform with Linux package naming, a dash can be added if word separation is important there. A placeholder email was inserted to satisfy the deb requirements for an author/maintainer email, this should be changed to either the author's email or a SideQuest support email.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-    "name": "SideQuest",
```
[the remark this answers is not in the document]
Comment (josephsmendoza) [reply package.json]: I'm guessing this changes the windows start menu entry? I hadn't tested that. I'll update to undo this change

File: package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+    "name": "sidequest",
```
Comment (josephsmendoza) [inline package.json]:
```suggestion
    "name": "SideQuest",
```

File: package.json [hunk 3]
```diff
@@ -32,8 +32,13 @@
...
             "target": [
-                "tar.xz"
-            ]
+                "tar.xz",
+                "AppImage",
+                "deb",
+                "rpm",
+                "pacman"
+            ],
+            "executableName":"sidequest"
```
[the remark this answers is not in the document]
Comment (josephsmendoza) [reply package.json]: unfortunately not, this only affects the launch command used in Linux but I haven't found documentation on changing the package name from the product name

**Comment by josephsmendoza** [comment]
It seems the atom text editor removes trailing spaces, resulting in a lot of insignificant lines in the diff.

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (josephsmendoza) [reply package.json]: that is, the binary is already lowercase, making this line necessary when the product name is not

**Comment by josephsmendoza** [comment]
Found the solution to Linux package naming at electron-userland/electron-builder#4037

**Comment by shaneharris** [comment]
We have decided not to go ahead with this as it adds additional support requirement. We invite you to fork and add these in. Thanks.

**Pull request closed without merging**